src/snippets/jss.js

```diff
@@ -0,0 +1,190 @@
+import React, { Component } from 'react';
+import jss from 'jss';
+import preset from 'jss-preset-default';
+import Logo from './logo';
+
+jss.setup(preset());
+
+const styles = {
+  container: {
+    display: 'flex',
+    flexDirection: 'column',
+    minHeight: '100%',
+    width: '100%',
+    backgroundColor: '#f6f9fc'
+  },
+  header: {
+    display: 'flex',
+    flexDirection: 'column',
+    height: '40%',
+    minHeight: 250,
+    width: '100%',
+    position: 'relative',
+    justifyContent: 'center'
+  },
+  logoContainer: {
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    height: 60,
+    width: 60,
+    borderRadius: 60,
+    backgroundColor: 'white',
+    boxShadow: { x: 0, y: 1, blur: 6, color: 'rgba(0, 0, 0, 0.1)' },
+    position: 'absolute',
+    top: '1rem',
+    left: '1rem',
+    zIndex: '2'
```

Comment:
btw. you can use a number here

Reply:
Will fix :) Good catch, and thank you again for the help!

Reply:
Feel free to open up an issue (or submit a PR :) ) when styled-jss fixes the React 16 issues!